transit/transit.go

```diff
@@ -195,11 +195,11 @@ type TypedValue struct {
 	// DoubleValue: A 64-bit double-precision floating-point number. Its
 	// magnitude is approximately &plusmn;10<sup>&plusmn;300</sup> and it
 	// has 16 significant digits of precision.
-	DoubleValue float64 `json:"doubleValue,omitempty"`
+	DoubleValue float64 `json:"doubleValue"`
```

Comment:
Ensure this has no side-effects in Data Geyser (same for IntegerValue)

Reply:
Shouldn't effect on data geyser because 'json' annotation used only in Go